moz_sql_parser/sql_parser.py

```diff
@@ -266,7 +266,7 @@ def to_string(instring, tokensStart, retTokens):
     (Literal("-")("op").setDebugActions(*debug) + expr("params")).addParseAction(to_json_call) |
     sqlString.setName("string").setDebugActions(*debug) |
     (
-        Word(alphas)("op").setName("function name").setDebugActions(*debug) +
+        Word(alphas + "_" + alphanums)("op").setName("function name").setDebugActions(*debug) +
```

Comment:
Does 

    ident("op").setName("function name").setDebugActions(*debug) +

work instead?

Reply:
Yes, it works, and that will be more common.you do great job!